src/server.js

```diff
@@ -31,7 +31,25 @@ app.post('/api/chat', async (req, res) => {
   }
 });
 
+const CHAT_USER_TABLE = "chat_user";
 
+app.post('/user', async (req, res) => {
```

Comment:
Is this end point only to create new users (sign up)?
If so maybe better to change the end point to something explanatory such as "create-user" or "signup".
Otherwise, it looks good to me!! :)